src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerIndividualValidator.java

```diff
@@ -96,21 +97,26 @@ private boolean validateDateOfBirth(LocalDate dateOfBirth, Errors errors, String
 
     private Errors validateNationality(String nationality, Errors errors, String loggingContext) {
         String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.BENEFICIAL_OWNERS_INDIVIDUAL_FIELD, BeneficialOwnerIndividualDto.NATIONALITY_FIELD);
-        nationalityValidator.validateAgainstNationalityList(qualifiedFieldName, nationality, errors, loggingContext);
+        boolean nationalityIsPresent = StringValidators.isNotBlank(nationality, qualifiedFieldName, errors, loggingContext);
+        if (nationalityIsPresent) {
+            nationalityValidator.validateAgainstNationalityList(qualifiedFieldName, nationality, errors, loggingContext);
+        }
         return errors;
     }
 
     private Errors validateSecondNationality(String nationality, String secondNationality, Errors errors, String loggingContext) {
         String qualifiedFieldNameFirstNationality = getQualifiedFieldName(OverseasEntitySubmissionDto.BENEFICIAL_OWNERS_INDIVIDUAL_FIELD, BeneficialOwnerIndividualDto.NATIONALITY_FIELD);
         String qualifiedFieldNameSecondNationality = getQualifiedFieldName(OverseasEntitySubmissionDto.BENEFICIAL_OWNERS_INDIVIDUAL_FIELD, BeneficialOwnerIndividualDto.SECOND_NATIONALITY_FIELD);
-        nationalityValidator.validateSecondNationality(
-                qualifiedFieldNameFirstNationality,
-                qualifiedFieldNameSecondNationality,
-                nationality,
-                secondNationality,
-                errors,
-                loggingContext);
-
+        boolean isNotBlank = StringUtils.isNotBlank(secondNationality);
```

Comment:
i'd rename the isNotBlank as it doesn't say what is blank, something like isSecondNationalityNotBlank would be better

Reply:
Not sure we really need to as it is in a method that names the field but have changed it.